codespell_lib/data/dictionary.txt

```diff
@@ -3045,6 +3047,8 @@ emblamatic->emblematic
 eminate->emanate
 eminated->emanated
 emision->emission
+emiss->remiss, amass, 
```

Comment:
can also be `amiss`

Reply:
right! btw, didn't find that the word `amissed` exists, so I didn't touch the next line